<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WReportBuilder</class>
 <widget class="QWidget" name="WReportBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QListWidget" name="lwNames">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Columns</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="6">
             <widget class="QListWidget" name="lwCols">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Sum for</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="btnExtractCols">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resort/res.qrc">
                <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="btnAddFilter_2">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resort/res.qrc">
                <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="5">
             <widget class="QLineEdit" name="leSum"/>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnRemoveFilter_2">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resort/res.qrc">
                <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Subtotals</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="5">
             <widget class="QLineEdit" name="leSubtotals"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="1">
             <widget class="EQLineEdit" name="leColTitle">
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="EQLineEdit" name="leColWidth">
              <property name="inputMask">
               <string>###</string>
              </property>
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EQLineEdit" name="leColField">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>85</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Field</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="2" column="0" colspan="3">
             <widget class="QListWidget" name="lwFilters">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Filters</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btnAddFilter">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resort/res.qrc">
                <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnRemoveFilter">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resort/res.qrc">
                <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="3" column="0">
             <widget class="QLabel" name="ddfas">
              <property name="text">
               <string>Build</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>85</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="EQLineEdit" name="leFilterField">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Field</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="EQComboBox" name="cbFilterType">
              <item>
               <property name="text">
                <string>COMBOBOX</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>INTEGER</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>STRING</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DATE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COMBO MONTH</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="EQLineEdit" name="leFilterMark">
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Data type</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lbMark">
              <property name="text">
               <string>Filter values
SQL string</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="EQLineEdit" name="leFilterTitle">
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="EQLineEdit" name="leFilterBuild">
              <property name="AlwaysUpper" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="EQLineEdit" name="leFilterExprEmpty"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Expresion
if empty</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Query</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="teSQL"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="EQLineEdit" name="leCode">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="EQLineEdit" name="leName">
           <property name="AlwaysUpper" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="EComboBoxCompleter" name="cbMenu">
           <property name="SQL" stdset="0">
            <string>select f_id, f_name from serv_reports_group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEditGrup">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resort/res.qrc">
             <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Menu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbMainMenu">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reservation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reception</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cashier</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>City Ledger</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bookkeeping</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Restaurant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Analytics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Other Reports</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Access</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="EQLineEdit" name="leGroupAccess"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>81</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_7" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnNew">
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/revive.png</normaloff>:/images/revive.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>542</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>EComboBoxCompleter</class>
   <extends>QComboBox</extends>
   <header location="global">ecomboboxcompleter.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leColField</tabstop>
  <tabstop>leColTitle</tabstop>
  <tabstop>leColWidth</tabstop>
  <tabstop>leFilterField</tabstop>
  <tabstop>leFilterTitle</tabstop>
  <tabstop>cbFilterType</tabstop>
  <tabstop>leFilterBuild</tabstop>
  <tabstop>leFilterMark</tabstop>
  <tabstop>leCode</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>cbMenu</tabstop>
  <tabstop>btnEditGrup</tabstop>
  <tabstop>leGroupAccess</tabstop>
  <tabstop>btnAddFilter_2</tabstop>
  <tabstop>btnRemoveFilter_2</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>lwNames</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>lwFilters</tabstop>
  <tabstop>btnAddFilter</tabstop>
  <tabstop>btnRemoveFilter</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnExtractCols</tabstop>
  <tabstop>lwCols</tabstop>
  <tabstop>teSQL</tabstop>
  <tabstop>leSum</tabstop>
 </tabstops>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
